example.env

```diff
@@ -8,3 +8,7 @@ PULUMI_SECRETS_PROVIDER={PULUMI_SECRETS_PROVIDER}
 # Used with "passphrase" secrets provider.
 # See https://www.pulumi.com/docs/reference/cli/environment-variables/.
 PULUMI_CONFIG_PASSPHRASE={PULUMI_CONFIG_PASSPHRASE}
+
+# These are used for the WCP telemetry functionality
+WCP_APP_URL=https://app.webiny.com/
+WCP_API_URL=https://d16ix00y8ek390.cloudfront.net
```

Comment:
Yesterday I mentioned we should double check these URLs - https://webiny.slack.com/archives/D02ADBU9URW/p1647984356070969?thread_ts=1647971580.397719&cid=D02ADBU9URW

I checked it now. If we were to open https://app.webiny.com/ and check dev tools / network tab, we'd see it's a different URL.

Also, as mentioned here, please update the doc with the correct URL. The ID in the URL is actually `d3mudimnmgk2a9`.

Please update the Notion doc with the correct URL while we're at it: https://www.notion.so/webiny/WCP-URLs-ae0b3f276fe94532a1ad2c8b38ef4987